src/requirements/admin/semester-season-migration.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+
+import { usernameCollection, semestersCollection } from '../../firebase-admin-config';
+
+/**
+ * Perform migration of semester type to semester season
+ */
+async function runOnUser(userEmail: string, runOnDB: boolean) {
+  const semesters = await semestersCollection
+    .doc(userEmail)
+    .get()
+    .then(it => {
+      const data = it.data();
+      return (data && data.semesters) || [];
+    });
+
+  console.group(`Running on ${userEmail}...`);
+  const newSemesters = semesters.map(semester =>
+    // each semester should have type but not necessarily season
+    ({ ...semester, season: semester.season || semester.type })
```

Comment:
maybe also do some cleanup to remove the old type field?

Reply:
There's currently still a dependency on `type` in the code. There was probably a better way to do it 😭 but I plan on doing the cleanup in a future script.

Reply:
Oh, I remember why I did it this way. It was because I didn't want to update the database schema while prod and other devs were still using `type`, so I split up the migration into multiple parts